### PR TITLE
Update membershipOutlierInsight.md

### PR DESCRIPTION
**Pull request by shresthabuddha**
Added property lastModifiedById and lastModifiedBy to track who modified the record. This will be the guid of user who modified the record and would be a directoryObject.

ADO Work item: 72752

### CODE REVIEW

File: api-reference/beta/resources/membershipOutlierInsight.md [hunk 1]
```diff
@@ -23,6 +23,7 @@ Inherits from [governanceInsight](governanceinsight.md).
 | Property    | Type   | Description |
 | :---------------| :---------- | :---------- |
 | memberId | String | Indicates the identifier of the user. |
+| lastModifiedById | String | Indicates the identifier of the user who modified the record. |
```
Comment (shubhamguptacal) [inline api-reference/beta/resources/membershipOutlierInsight.md:26]: Is this field nullable?

**Comment by Lauragra** [comment]
@shresthabuddha , it looks like this PR hasn't had a content review. Adding @FaithOmbongi . We'll need a changelog entry to call out the new properties.

**Comment by FaithOmbongi** [comment]
Hi @shresthabuddha - I see the changes in this PR aren't yet in the public schema.  To allow us to schedule this work for review, create a content review work item at https://aka.ms/msgraph-identity-workitem and add the links to the API.md and Schema PR in the work item description.

Also, add the changelog entry to this PR. You'll get the changelog stubs from your schema PR's build artifacts.

Hi @msewaweru - kindly assist to review this PR when it's ready to. Thanks.

File: api-reference/beta/resources/membershipOutlierInsight.md (hunk 1 above)
Comment (shresthabuddha) [reply api-reference/beta/resources/membershipOutlierInsight.md:26]: it will not be null as a default value will be added to it.

**Comment by msewaweru** [comment]
@shresthabuddha, please mark PR ready for review once you have added the change log. Thanks.

File: api-reference/beta/resources/membershipOutlierInsight.md [hunk 2]
```diff
@@ -33,6 +33,8 @@ Inherits from [governanceInsight](governanceinsight.md).
 |:---|:---|:---|
 |container|[directoryObject](../resources/directoryobject.md)| Navigation link to the container directory object. For example, to a group.|
 |member|[directoryObject](../resources/directoryobject.md)| Navigation link to a member object. For example, to a user.|
+|lastModifiedBy|[directoryObject](../resources/directoryobject.md)| Navigation link to a member object who modified the record. For example, to a user.|
```
Comment (msewaweru) [inline api-reference/beta/resources/membershipOutlierInsight.md:36]: Schema indicates the return type to be user not directoryobject. Please update.

**Review by msewaweru — CHANGES_REQUESTED** [review]
Hi @shresthabuddha, please check the return type of the new relationship as per the schema.

Also, add the change log for this update.

**Comment by shresthabuddha** [comment]
> Hi @shresthabuddha, please check the return type of the new relationship as per the schema.
> 
> Also, add the change log for this update.

@msewaweru added changelog + changed relationship type. Please check. Thank you for comment.

File: changelog/Microsoft.AAD.Reporting.json [hunk 3]
```diff
@@ -1,5 +1,23 @@
 {
   "changelog": [
+    {
+      "ChangeList": [
+        {
+          "Id": "d46f7136-1ef8-47e4-b10e-8e31a87d47f4",
+          "ApiChange": "Resource",
```
Comment (msewaweru) [inline changelog/Microsoft.AAD.Reporting.json:7]:
```suggestion
          "ApiChange": "Relationship",
```

File: changelog/Microsoft.AAD.Reporting.json [hunk 4]
```diff
@@ -1,5 +1,23 @@
...
+          "ChangedApiName": "lastModifiedBy",
+          "ChangeType": "Addition",
+          "Description": "Added the **lastModifiedBy** relationship to the [membershipOutlierInsight](https://learn.microsoft.com/en-us/graph/api/resources/membershipOutlierInsight?view=graph-rest-beta) resource.",
```
Comment (msewaweru) [inline changelog/Microsoft.AAD.Reporting.json:10]:
```suggestion
          "Description": "Added the **lastModifiedBy** navigation property to the [membershipOutlierInsight](https://learn.microsoft.com/en-us/graph/api/resources/membershipOutlierInsight?view=graph-rest-beta) resource.",
```

File: api-reference/beta/resources/membershipOutlierInsight.md [hunk 5]
```diff
@@ -33,6 +33,8 @@ Inherits from [governanceInsight](governanceinsight.md).
 |:---|:---|:---|
 |container|[directoryObject](../resources/directoryobject.md)| Navigation link to the container directory object. For example, to a group.|
 |member|[directoryObject](../resources/directoryobject.md)| Navigation link to a member object. For example, to a user.|
+|lastModifiedBy|[directoryObject](../resources/user.md)| Navigation link to a member object who modified the record. For example, to a user.|
```
Comment (msewaweru) [inline api-reference/beta/resources/membershipOutlierInsight.md:36]:
```suggestion
|lastModifiedBy|[user](../resources/user.md)| Navigation link to a member object who modified the record. For example, to a user.|
```

**Review by msewaweru — CHANGES_REQUESTED** [review]
@shresthabuddha - thanks for the updates. A few final comments:

- Suggested that new relation return type be user instead of directoryObject as per the schema.
- The API change in the changelog should be relationship instead of resource.

File: changelog/Microsoft.AAD.Reporting.json [hunk 6]
```diff
@@ -1,5 +1,23 @@
...
+          "Target": "membershipOutlierInsight"
+        }
+      ],
+      "Id": "d46f7136-1ef8-47e4-b10e-8e31a87d47f4",
+      "Cloud": "Prod",
+      "Version": "beta",
+      "CreatedDateTime": "2023-03-15T22:19:06.7781678Z",
+      "WorkloadArea": "Reports and audit",
```
Comment (msewaweru) [inline changelog/Microsoft.AAD.Reporting.json:18]:
```suggestion
      "WorkloadArea": "Reports",
```

**Review by msewaweru — CHANGES_REQUESTED** [review]

**Comment by shresthabuddha** [comment]
> @shresthabuddha - thanks for the updates. A few final comments:
> 
> * Suggested that new relation return type be user instead of directoryObject as per the schema.
> * The API change in the changelog should be relationship instead of resource.

Thank you @msewaweru for the suggestions. Applied all of them.

**Review by msewaweru — APPROVED** [review]
Thanks, @shresthabuddha for the updates. LGTM!

**Pull request merged**